rwengine/src/engine/GameState.hpp

```diff
@@ -243,6 +243,13 @@ struct GarageInfo {
     }
 };
 
+enum HudFlash {
```

Comment:
This could be ``enum class HudFlash``, as you're already referencing the values by their enum scope.